rfcs/text/0000-conceptual-data-model.md

```diff
@@ -0,0 +1,109 @@
+
+![Figure 3: MISSING](../images/0000-conceptual-data-model/new-analysis-graph.png)
+
+Figure 3: Graph with a new analysis:  In this case, a second, independent analysis is run.  This might be the case when a new different analysis algorithm is being used.  The head graph includes both analysis results.   The provenance graph is obtained by reverse traversing the edges shown in red.
+
+
+![Figure 4: MISSING](../images/0000-conceptual-data-model/reanalysis-graph.png)
+
+Figure 4: Graph with an upstream consequential metadata change that triggers a reanalysis.  This is the case where there is an error in a metadata field that is used to parameterize the analysis. Here the sequencing didn't change, however, the downstream analysis was triggered by a change in the biomaterial metadata (B1).  The provenance graph reverse traverses the edges shown in red, going to B1.2 even though the process metadata P1.1 has not changed.
+
+
+![Figure 5: MISSING](../images/0000-conceptual-data-model/amended-graph.png)
+
+Figure 5: Graph with an upstream inconsequential metadata change that does not trigger a reanalysis.  Correcting natural-language descriptions or migrating the metadata schema are examples of this type of update.  Reverse-traversing the red edges gives the amended provenance graph.
+
+The *Project* metadata entity is as a container for all metadata and data that is part of the project.  All data in the DCP belongs to at least one project.  A given metadata instance may be associated with more than one project, such as a sample sequenced by multiple projects. Derived projects can be created for an analysis that used data from multiple projects.
+
+### Example use cases
+These sections collect various use cases to illustrate how they are represented or are problematic for this model.
+
+- Contributor discovers the wrong FASTQ was uploaded or a FASTQ is missing.  An update to the primary bundle is submitted with a replacement or new FASTQ.  This is a consequential change to analysis and will result in a new provenance graph.
+
+- Change in B1 in a metadata field that is used to parameterize the analysis so analysis is triggered but the outputs (F3) from analysis & re-analysis checksums are identical.  It would be desirable for the user to know the results are not changed with the new parameters.  This is not currently representable.  The users would have to detect themselves.
+
+- The similar case of updated software or parameters (P2) causes re-analysis but the outputs have identical checksums.
+
+
+### Unresolved Questions
```

Comment:
Scenario when 1 primary bundle is ingested & released, then it's later discovered that it should be split into multiple primary bundles. Seems like this should create brand new bundles, rather than try to pick 1 to be the new version.

Reply:
Then you lose history. I think this is an argument for seriously considering whether history should live in metadata rather than be conveyed in bundle versions. More details will be in [this DCP session](https://docs.google.com/spreadsheets/d/1ZUVm-bt6hj7IqffhmXBB8vZNQ4rOQrL0mpo958_tTmQ/edit#gid=0&range=C4) if it's accepted and I have time to create a presentation.